### PR TITLE
[Submission] Manilow

### PR DESCRIPTION
**Pull request by ethman**
**Ethan Manilow**

Dear mdx workshop committee,

I request a review for the following MDX submission:

### **Title: Source Separation: Explorations and Applications**  

**Author(s): Ethan Manilow**  

**Category ***

* [ ] `POSTERS` _(+Abstract: 250 Words+one page pdf)_
  > As a participant of the MDX challenge, please know that your poster would be automatically accepted after a minimal prescreening. Hence, don't forget to provide us with the link to your team in the AIcrowd website.
  > You are also welcome to submit us some unrelated abstract, in case your interest would be in presenting some method or idea you would just like to discuss with the participants passing by.  
  > => You *don't* need to include your poster in the PR. Just be ready to have it for D-day and show it to the audience through screen sharing.
* [x] `LONG TALK`
  > A long talk will last about 20-30min, including questions. This can be about some recent paper you published, or may also present some original research and investigations you did. If you pick this category, please provide us with an extended abstract of the talk, or directly with the slides if you have them available, even in an unfinished state, at your convenience. 
* [ ] `DISCUSSIONS`
  > This category is intended to stimulate fruitful discussions and collaborations. If you pick it, it means you are willing to chair some open discussion with the participants, mainly around one particular aspect of music demixing research. For that purpose, you should provide a minimal abstract as well as a list of a few questions you would like the participants to discuss about. Don't hesitate to propose ideas, We will possibly contact you in case we identify several similar topics that were proposed!

**Preferred Timezone***

* [ ] A 11:00 PM (PDT) to 9:00 AM PDT
* [x] B 11:00 AM (PDT) to 9:00 PM PDT

  > The preferred timezone will help us to organize the day schedule. You should select at least one, but can perfectly select both if you don't care. We will organize the programme so that to maximally fit your preferences.
---

**ORGANISATION COMMITTEE (do not fill out)**

* [x] Editor acknowledgment
* [x] Reviewer 1 
* [ ] Reviewer 2
* [x] Review 1 decision [accept/reject]
* [ ] Review 2 decision [accept/reject]
* [ ] Editor decision [accept/reject]

### CODE REVIEW

File: paper.md [hunk 1]
```diff
@@ -1,129 +1,89 @@
...
+the vast potential that lays relatively untapped in these large pretrained
+models.
+
+The second part will focus on newly added deep learning
+capabilities in the Audacity audio editor, which simplifies the process of
+getting trained models into the hands of end-users. This work lets model
+creators a way to sidestep DAW-specific development work and enables them to
+upload pretrained PyTorch models to HuggingFace, where they will automatically
+be discoverable and runnable within Audacity's UI by end-users. In this talk,
+I will provide a high level overview of this software framework. We hope
```
Comment (aliutkus) [inline paper.md:41]: unfinished

File: paper.md [hunk 2]
```diff
@@ -1,129 +1,89 @@
...
-problems it aims to solve.
-* For all categories except discussions, please also briefly summarize the contributions.
+This talk will consist of two parts, the first of which outlines
+exploratory work in using pretrained, general purpose music models for source
+separation. I will discuss an unsupervised method that uses
+large, pretrained music models for audio-to-audio tasks–like source separation
+and style transfer; all without any retraining. Inspired by the popular
+VQGAN+CLIP combination for making generative visual art, I accomplish audio
+tasks by pairing OpenAI's Jukebox with a pretrained music tagger in a system.
+called TagBox. I will showcase some fun and interesting results, contextualiz
```
Comment (aliutkus) [inline paper.md:30]: contextualize

**Review by aliutkus — COMMENTED** [review]
can't wait to listen to this, it's great. thanks for the submission

**Review by aliutkus — COMMENTED** [review]
This talk looks very nice. I wonder whether you will include some material for the audience like demos / code etc. I think it could be great to include that as references in this paper

**Comment by aliutkus** [comment]
To all participants:
please note that you should update your submission by November 6th 23:00 UTC

Thank you very much

**Comment by ethman** [comment]
Hello MDX organizers. This seems to have the "Accepted" tag added to it, but I don't see a schedule anywhere. When should I expect to be speaking?

**Comment by faroit** [comment]
@ethman sorry Ethan, I did only had time to add the A schedule to midspace today. Your talk is scheduled to 11/13/2021 3:10:00 UTC. Is that okay?

**Comment by ethman** [comment]
Hi @faroit, at this point I have things scheduled starting at 6PM US Central time this Friday. I believe that is anytime before Nov 13, 23:59:59 UTC. I can start any time before then.

**Comment by faroit** [comment]
@ethman not sure if I understand correctly. The B sessions of the MDX workshop start at 6PM CST, so that would make it difficult to start earlier. Would you able to provide a recording and answer q&a in midspace later?

**Comment by ethman** [comment]
Ah, sorry! I can't see the B session talks on MidSpace yet, so I didn't know when they were happening. I can probably maneuver some things; would it be possible to get a slot starting as close to 6PM CST as possible? Sorry to jumble your schedule up, but with virtual conferences I also have to juggle other aspects of life that are going on!

**Comment by faroit** [comment]
👏 The submission has been published on midspace (the ISMIR conference platform). The submissions will also be hosted on the workshop website tomorrow. 

The schedule can be seen there or [here](https://docs.google.com/spreadsheets/d/1PaksFcHWUfNPMg-XloA3CJtrvb9uMU0erCFVyvFi9B4/edit?usp=sharing). Please let us know if there is anything missing. Otherwise we see each other on Friday/Saturday!

**Comment by faroit** [comment]
Hi @ethman , sorry for the long waiting time but I wasn't able to continue working on the proceedings before. 

I managed to finally work on cutting the video recordings of the workshop and plan to upload the recordings to YouTube very soon. In order to allow me to do this, please acknowledge the following:  

> I hereby authorize the right and permission to copyright and/or publish, reproduce or otherwise use my name, voice, and audio-visual recordings. I acknowledge and understand these materials about or of me may be used for both commercial and/or non-commercial purposes.

> I understand that my image may be edited, copied, exhibited, published and/or distributed. There is no time limit on the validity of this release nor are there any geographic limitations on where these materials may be distributed.

> I authorize that the video will be made available under the [CC BY-SA 4.0 license](https://creativecommons.org/licenses/by-sa/4.0/) on the conference website.

and comment on this by replying with `acknowledge`.

**Pull request closed without merging**